RosBridgeClient/Message.cs

```diff
@@ -15,6 +15,7 @@
 limitations under the License.
 */
 
+
```

Comment:
Please add the comment to your changes as I requested in your previous pull request.